scripts/utils/skill-logger.sh

```
@@ -0,0 +1,57 @@
+# Args: $1=skill_name, $2=action, $3=exit_code (optional, defaults to $?)
+log_skill_invocation() {
+  local skill=$1
+  local action=$2
+  local exit_code=${3:-$?}
+
+  # Calculate duration
+  local end_time=$(python3 -c 'import time; print(int(time.time() * 1000))' 2>/dev/null || echo $(($(date +%s) * 1000)))
+  local duration=$((end_time - START_TIME))
+
+  # Get current timestamp
+  local timestamp=$(date -u +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)